packages/help-center/src/components/help-center-content.tsx

```diff
@@ -35,6 +35,23 @@ interface ProtectedRouteProps {
 	children: React.ReactNode;
 }
 
+// Prevent not eligible users from accessing odie/wapuu.
+const ProtectedRoute: React.FC< ProtectedRouteProps > = ( {
```

Comment:
This is the component that was within a component which was causing the extra rendering, correct?

Reply:
Yes. A component should not be defined inside another component.

When a component renders, each time it renders the control will be redefined and recreated, causing re renders when it shouldn't be rendered. This component in particular does not have useEffect or useState, if it would have it, the error would be more evident, as it couldn't keep the state between renders.